src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-dialog/hw-passphrase-dialog.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { HwPassphraseDialogComponent } from './hw-passphrase-dialog.component';
+
+describe('HwPassphraseDialogComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
describe("HwPassphraseDialogComponent", () => {
```